### DISCUSSION TITLE
服務學習這門課程

### DISCUSSION THREAD

**Discussion by 113607514** [1101-GS4719-C/Python-Community-GS4719-C#3686] [General]
服務學習這門課程對我來說，不僅僅是一項學業要求，更是一段深刻的自我反思和成長過程。當我第一次參與服務學習時，我帶著一絲好奇和不安，總覺得自己做得還不夠好，甚至質疑這樣的努力是否能帶來真實的改變。然而，隨著時間的推移，我開始理解服務學習的真正意義。每一次與需要幫助的群體互動，每一次自己付出的努力，都讓我更加珍惜社會中每個不被重視的角落，並意識到自己的行動如何影響他人的生活。這門課程不僅讓我學會了如何將理論與實踐結合，還讓我學會了同理心、責任感和對社會的深度關懷。它讓我明白，真正的學習，不僅來自書本知識的累積，還來自於服務他人和反思自己在其中的角色。

**Comment by Mminniehuang** [comment]
談到服務學習，我就想到這學期修的一門課程中，在講到價值理性與工具理性的時候，批判服務學習時數的設立的討論。
我覺得服務學習時數的立意良好，但是因為把它作為免試入學的比序分數項目之一啦、畢業門檻啦，其實多數人都是敷衍被動的。當然可能會覺得說至少比沒有要求的情況根本沒有人去做來的好，但我在想這應該是教育體制、社會文化、人際互動模式改變等等的影響了。

**Comment by poyen0806** [comment]
我羞的服務學習還不錯，先去辦營隊帶國中小生寫，讓他們還沒學英文就先學寫code，體驗人生辛苦，再去寫服學相關的網頁，奠定我寫網頁及app的基礎。我覺得服務學習不錯，比很多課好很多了。

**Comment by yue0903ZZ** [comment]
我們系的服務學習很廢，要重新規劃系上垃圾間，做給高中生看的網頁。
服務學習零學分，有過就好了，所以我上課都在發呆。
我的室友aka未來機械系卷哥許翊軒就蹺很多堂課，還沒有被當。
這學期還不選服務學習，跑去上靜材，太電了啦。
